components/dashboard/sections/transactions-imports/TransactionsImport.tsx

```diff
@@ -373,7 +390,7 @@ export const TransactionsImport = ({ accountSlug, importId }) => {
                       <StepMapCSVColumns
                         importId={importId}
                         file={csvFile}
-                        currency={importData.account.currency}
+                        currency={importData.account['currency']}
```

Comment:
Could the currency be added to the account field of `transactionsImportQuery`?

Reply:
100%. This was a quick workaround that shouldn't have been left there.